### PR TITLE
chore(1602-fix) - Setting size as 4096

### LINKED ISSUE #1602: Add xvfb in CI (test-and-build) to run axe test cases

### PR DESCRIPTION
**Pull request by hybridx**
# Closes/Fixes/Resolves

### CODE REVIEW

File: packages/lighthouse-spa/src/environments/environment.prod.ts [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+   WS_URL: 'ws://localhost:8080/api/graphql/subscriptions',
+   LH_SERVER_URL: 'http://test.lhci.redhat.com',
+   LH_CONTACT_MAIL: 'one-devel@redhat.com',
+   LH_DOC_URL: 'undefined',
+   OPCBASE_API_BASE_PATH: 'http://localhost:8080/api/graphql',
+   OPCBASE_SUBSCRIPTION_BASE_PATH: 'ws://localhost:8080/api/graphql/subscriptions',
+   OPCBASE_KEYCLOAK_URL: 'https://auth.test.com/auth',
+   OPCBASE_KEYCLOAK_CLIENT_ID: 'one' ,
+   OPCBASE_KEYCLOAK_REALM: 'EmployeeIDP',
+   SENTRY_DSN: undefined,
```
Comment (riginoommen) [inline packages/lighthouse-spa/src/environments/environment.prod.ts:14]: Please remove this values

File: packages/search-spa/src/environments/environment.prod.ts [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
-  OPCBASE_KEYCLOAK_REALM: '',
-  SENTRY_DSN: undefined,
+   production: true,
+   API_URL: 'http://localhost:8080/api/graphql',
+   OPCBASE_API_BASE_PATH: 'http://localhost:8080/api/graphql',
+   OPCBASE_SUBSCRIPTION_BASE_PATH: 'ws://localhost:8080/api/graphql/subscriptions',
+   OPCBASE_KEYCLOAK_URL: 'https://auth.test.com/auth',
+   OPCBASE_KEYCLOAK_CLIENT_ID: 'one' ,
+   OPCBASE_KEYCLOAK_REALM: 'EmployeeIDP',
+   SENTRY_DSN: undefined,
```
Comment (riginoommen) [inline packages/search-spa/src/environments/environment.prod.ts:10]: Please remove this value

**Review by riginoommen — CHANGES_REQUESTED** [review]

File: packages/lighthouse-spa/src/environments/environment.prod.ts (hunk 1 above)
Comment (hybridx) [reply packages/lighthouse-spa/src/environments/environment.prod.ts:14]: Got it, Thanks

File: packages/search-spa/src/environments/environment.prod.ts (hunk 2 above)
Comment (hybridx) [reply packages/search-spa/src/environments/environment.prod.ts:10]: Got it, Thanks

**Review by riginoommen — DISMISSED** [review]

**Review by riginoommen — APPROVED** [review]

**Review by riginoommen — APPROVED** [review]

**Review by riginoommen — DISMISSED** [review]

**Comment by hybridx** [comment]
Support for web-test-runner to be added later with nx cloud or self hosted runner

**Review by deshmukhmayur — APPROVED** [review]

**Review by deshmukhmayur — CHANGES_REQUESTED** [review]
@hybridx can you clean up the commits?
Merge them into logical commits.

**Comment by hybridx** [comment]
> @hybridx can you clean up the commits? Merge them into logical commits.

@deshmukhmayur I think that we can squash these commits, they are changes to only CI/CD mainly

**Review by deshmukhmayur — APPROVED** [review]

**Review by riginoommen — APPROVED** [review]

**Pull request merged**